docs/manual/other-topics/extending-data-types.md

```diff
@@ -0,0 +1,113 @@
+# Extending Data Types
```

Comment:
Many users reported that custom datat-ypes doesn't work. Have you tried this @papb?

Reply:
It doesn't work, indeed. I was a bit unsure of what to do with this. In the end, I considered the following:

* What I am doing here is just reorganizing the docs
* This content [already exists in the current docs](https://sequelize.org/master/manual/data-types.html#extending-datatypes), even though it doesn't work
* So I decided to reorganize the code *as if it worked*, leaving the actual matter of fixing it to the other issues which revolve around it

I thought this was "less worse" than just removing it from the docs. It is a known issue, and hopefully we can address it in the near future :)

And this way, when it gets fixed, the documentation around it will be much easier to update (if necessary!)

So in the end I think it's okay to leave it this way, what do you think?

Reply:
Yeah we need to fix that in near future, keep these docs for now